src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -69,6 +71,13 @@ type Config struct {
 
 // StartPolling starts the process of polling for new configuration files.
 func StartPolling() {
+	// Force detour to whitelist chained domain
+	u, err := url.Parse(chainedCloudConfigUrl)
+	if err != nil {
+		log.Fatalf("Unable to parse chained cloud config URL: %v", err)
```

Comment:
Well, this means that our code is fundamentally broken.  It's not like it'll happen spuriously, and this way we find out during development rather than possibly missing it.